code-challenges.test.js

```diff
@@ -20,33 +20,361 @@ const secretCodeWord2 = "Gobbledygook"
+//         expect(valueContainsALetter(fruitArray, letterE)).toEqual(["Cherry", "Blueberry", "Peach"])
+//         expect(valueContainsALetter(fruitArray, letterM)).toEqual(["Mango"])
+//         expect(valueContainsALetter(fruitArray, letterY)).toEqual(["Cherry", "Blueberry"])
+//     })
+// })
+
+// jest-test-1:
+// FAIL  ./code-challenges.test.js
+// valueContainsALetter
+//   ✕ return an aray of all the words containing that particular letter (1 ms)
+
+// ● valueContainsALetter › return an aray of all the words containing that particular letter
+
+//   ReferenceError: valueContainsALetter is not defined
 
-const letterA = "a"
-// Expected output: ["Mango", "Apricot", "Peach"]
-const letterE = "e"
-// Expected output: ["Cherry", "Blueberry", "Peach"]
+//     161 | describe("valueContainsALetter", () => {
+//     162 |     it("return an aray of all the words containing that particular letter", () => {
+//   > 163 |       expect(valueContainsALetter(fruitArray, letterA)).toEqual(["Mango", "Apricot", "Peach"])
+//         |       ^
+//     164 |       expect(valueContainsALetter(fruitArray, letterE)).toEqual(["Cherry", "Blueberry", "Peach"])
+//     165 |       expect(valueContainsALetter(fruitArray, letterM)).toEqual(["Mango"])
+//     166 |       expect(valueContainsALetter(fruitArray, letterY)).toEqual(["Cherry", "Blueberry"])
+
+//     at Object.expect (code-challenges.test.js:163:7)
+
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
+
+// jest-test-2:
+// FAIL  ./code-challenges.test.js
+// valueContainsALetter
+//   ✕ return an aray of all the words containing that particular letter (1 ms)
+
+// ● valueContainsALetter › return an aray of all the words containing that particular letter
+
+//   expect(received).toEqual(expected) // deep equality
+
+//   Expected: ["Mango", "Apricot", "Peach"]
+//   Received: undefined
+
+//     161 | describe("valueContainsALetter", () => {
+//     162 |     it("return an aray of all the words containing that particular letter", () => {
+//   > 163 |       expect(valueContainsALetter(fruitArray, letterA)).toEqual(["Mango", "Apricot", "Peach"])
+//         |                                                         ^
+//     164 |       expect(valueContainsALetter(fruitArray, letterE)).toEqual(["Cherry", "Blueberry", "Peach"])
+//     165 |       expect(valueContainsALetter(fruitArray, letterM)).toEqual(["Mango"])
+//     166 |       expect(valueContainsALetter(fruitArray, letterY)).toEqual(["Cherry", "Blueberry"])
+
+//     at Object.toEqual (code-challenges.test.js:163:57)
+
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
+
+// jest-test-3:
+// PASS  ./code-challenges.test.js
+// valueContainsALetter
+//   ✓ return an aray of all the words containing that particular letter (1 ms)
+
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
 
 // b) Create the function that makes the test pass.
 
```

Comment:
I would take another look at this one. I copied it into my vscode and it is throwing an error

Reply:
Hey Nicole, on this one are you referring to the just the test? I copied it from here right into my vscode and it passed on my end.